lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -126,15 +127,16 @@ def instructor_dashboard_2(request, course_id):
 ## section_display_name will be used to generate link titles in the nav bar.
 
 
-def _section_e_commerce(course, access):
+def _section_e_commerce(course, access, is_white_label):
     """ Provide data for the corresponding dashboard section """
     course_key = course.id
     coupons = Coupon.objects.filter(course_id=course_key).order_by('-is_active')
-    course_price = None
+
+    course_price = 0
+    for mode in CourseMode.modes_for_course(course_key):
+        course_price = mode.min_price if mode.min_price > course_price else course_price
```

Comment:
I'm not sure I understand this change.  Why are we choosing the maximum price instead of enforcing the assumption that there's exactly one paid mode?

Also, this looks like it's going to hit the database again to retrieve the course mode information, which the caller already has.  Would it be possible to just pass the course mode into this function to avoid the second database query?